misc/buildtestdata.sh

```diff
@@ -33,7 +43,7 @@ mkdir -p "${E2EDATA}"
 
 (cd "${E2EDATA}" && tar -j -c -f net_done.tar.bz2 --exclude node.log --exclude agreement.cdv net)
 
-if false; then
```

Comment:
Sorry, I think I left this half done, there are two tests, `misc/e2etest.py` which is the older one and `misc/e2elive.py` which is newer. The 'live' test compares `algod` API output to Indexer API output and should be better and more sustainable. The older e2etest.py uses Python to pick apart the algod account database. The extra 320 rounds below are only needed for e2etest.py.

Reply:
See #264 
I hope to just switch over to the new test there.